src/drive/web/modules/viewer/FileOpenerExternal.jsx

```diff
@@ -75,8 +76,11 @@ export class FileOpener extends Component {
             <Viewer
               files={[file]}
               currentIndex={0}
-              onChange={doNothing}
-              onClose={withCloseButtton ? this.navigateToDrive : null}
+              onChangeRequest={doNothing}
+              onCloseRequest={withCloseButtton ? this.navigateToDrive : null}
+              renderFallbackExtraContent={file => (
+                <Fallback file={file} t={t} />
```

Comment:
Can we add a comment about why we need to pass `t`? It's pretty fresh now, in a few months I doubt